JapaneseFilter/sections/specific.txt

```diff
@@ -2390,6 +2390,7 @@ pref.aomori.lg.jp##.pr_banner
 pref.niigata.lg.jp###top_banner_koukoku
 pref.aichi.jp##.top_kokoku_box
 appbank.net##.appbank-tags > a.RWh2:first-child
+appbank.net##.entry-list-item-native-ad
```

Comment:
Add also `##.body-ad` seen in the same page as `.lets-push`.

Reply:
I see. Done